ui_framework/components/form/_select.scss

```diff
@@ -0,0 +1,12 @@
+/**
+ * 1. Embedded SVG of fa-caret-down (https://github.com/encharm/Font-Awesome-SVG-PNG/blob/master/black/svg/caret-down.svg).
+ * 2. Make room on right side for the caret.
+ */
+.kuiSelect {
+  @include formControl;
+  padding-right: 30px; /* 2 */
+  background-image: url('data:image/svg+xml;utf8,<svg width="1792" height="1792" viewBox="0 0 1792 1792" xmlns="http://www.w3.org/2000/svg"><path d="M1408 704q0 26-19 45l-448 448q-19 19-45 19t-45-19l-448-448q-19-19-19-45t19-45 45-19h896q26 0 45 19t19 45z"/></svg>'); /* 1 */
```

Comment:
Great questions... I'm not sure how browsers handle SVGs. Maybe it's possible that a browser or even the OS could cache the resulting image after rendering an SVG? But either way, I'd say let's optimize this once when encounter a performance problem.

A sprite might be better, but this lives within the UI Framework and currently the UI Framework only exports a single CSS file for our build system. We'd need to integrate this with our build system somehow, e.g. by exporting images in a way that optimize can copy and serve up from the appropriate location. So I don't think it's something we can easily do just now.